Cabal/src/Distribution/Simple/Setup/Config.hs

```diff
@@ -322,12 +322,7 @@ defaultConfigFlags progDb =
     , configCabalFilePath = NoFlag
     , configVerbosity = Flag normal
     , configUserInstall = Flag False -- TODO: reverse this
-#if defined(mingw32_HOST_OS)
-        -- See #8062 and GHC #21019.
-    , configGHCiLib = Flag False
-#else
-    , configGHCiLib = NoFlag
-#endif
```

Comment:
This seem fairly fragile, and again should be driven from the compiler, not the build tool.